### PR TITLE
Gantt view 2.0

### PR DESCRIPTION
**Pull request by mgeubelle**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: doc/reference/views.rst [hunk 1]
```diff
@@ -1216,56 +1216,72 @@ take the following attributes:
...
+``default_scale``
+  default scale when rendering the view. Possible values are (default: ``month``):
+
+  * ``day``
+  * ``week``
+  * ``month``
+  * ``year``
+
+``limit``
+  number of records that the gantt view will display at maximum (default: 200)
```
[the remark this answers is not in the document]
Comment (mge-odoo) [reply doc/reference/views.rst]: maybe consider to set draggable and reiszable on hover only

[the remark this answers is not in the document]
Comment (mge-odoo) [reply doc/reference/views.rst]: do not render pills (closed group by default)

**Comment by aab-odoo** [comment]
robodoo rebase-ff

**Comment by dmo-odoo** [comment]
@aab-odoo mge m'a dit que tu serais dispo pour r+ ? :)

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**